tests/integration/lsp_tests.rs

```diff
@@ -17153,3 +17153,38 @@ fn lsp_wasm_module() {
   );
   client.shutdown();
 }
+
+#[test]
+fn wildcard_augment() {
+  let context = TestContextBuilder::new().use_temp_cwd().build();
+  let mut client = context.new_lsp_command().build();
+  let temp_dir = context.temp_dir().path();
+  let source = source_file(
+    temp_dir.join("index.ts"),
+    r#"
+      import styles from "./hello_world.scss";
+
+      function bar(v: string): string {
+        return v;
+      }
+
+      bar(styles);
+    "#,
+  );
+  temp_dir.join("index.d.ts").write(
+    r#"
+      declare module '*.scss' {
+        const content: string;
+        export default content;
+      }
+    "#,
+  );
+  temp_dir
+    .join("hello_world.scss")
+    .write("body { color: red; }");
+
+  client.initialize_default();
+
+  let diagnostics = client.did_open_file(&source);
+  assert_eq!(diagnostics.all().len(), 0);
+}
```

Comment:
Should we `client.shutdown()` here for consistency? Probably fine if we don't though

Reply:
I think we could remove all of those. I've also just been keeping up the song and dance when writing those tests lol

Reply:
We should probably be testing in at least one if the process ends itself upon shutdown, but I think in all of these we're just forcefully ending the process after sending the notification..